### DISCUSSION TITLE
Duplicate the schema of a whole database. Is this a bug?

### DISCUSSION THREAD

**Discussion by 0x7FFFFFFFFFFFFFFF** [dataplat/dbatools#9210] [General]
I've created an empty database `aw2` and trying to copy the schema of `AdventureWorks2016` to this new database. Below is my commands. As you can see, it generates the error `Cannot use a CONTAINS or FREETEXT predicate on table or indexed view 'HumanResources.JobCandidate' because it is not full-text indexed`. Is this a bug?
```
PS C:\> New-DbaDbTransfer -SqlInstance node1 -DestinationSqlInstance node1 -Database adventureworks2016 -DestinationDatabase aw2 -CopyAllObjects -SchemaOnly -Verbose | Invoke-DbaDbTransfer -Verbose
VERBOSE: [23:52:40][Connect-DbaInstance] String is passed in, will build server object from instance object and other parameters, do some checks and then return the server object
VERBOSE: [23:52:40][Connect-DbaInstance] authentication method is 'local integrated'
VERBOSE: Performing the operation "Invoke-DbaDbTransfer" on target "Begin transfer".
WARNING: [23:52:44][Invoke-DbaDbTransfer] Transfer failed | Cannot use a CONTAINS or FREETEXT predicate on table or indexed view 'HumanResources.JobCandidate' because it is not full-text indexed.
PS C:\>
```

**Comment by wsmelton** [comment]
That is coming from code in the database. That error generally comes from a query query in a stored procedure or maybe view that has a `CONTAINS` reference which requires a Full Text index exist on the table being referenced. 

Since you are using Adventure Works I'd see if there is a Full Text index on the `JobCandidate` table.
